### PR TITLE
All changes missing Kevin's sink changes

### PR DESCRIPTION
**Pull request by faisalaltell12**
# Choose a PR Template

Switch to "Preview" on this description then select one of the choices below.

<a href="?expand=1&template=data_plane_template.md">Click here</a> to open a PR for a Data Plane API.

<a href="?expand=1&template=control_plane_template.md">Click here</a> to open a PR for a Control Plane (ARM) API.

### CODE REVIEW

**Review by kevinmichaelbowersox — APPROVED** [review]

File: specification/purview/data-plane/Azure.Analytics.Purview.Share/preview/2022-06-30-preview/examples/ReceivedShares_AdlsGen2AccountUpdate.json [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  "responses": {
+    "202": {
+      "headers": {
+        "Date": "Wed, 13 Sep 2017 18:04:32 GMT",
+        "x-ms-request-id": "d5496da4-9c52-402f-b067-83cc9ddea888",
+        "x-ms-correlation-request-id": "25c78f97-0b0a-4fe9-ad39-883a482265cd",
+        "x-ms-long-running-operation": true,
+        "Azure-AsyncOperation": "https://accountName.purview.azure.com/share/operationResults/dad6baec-3a39-41df-a469-843a9ee94213?api-version=2022-06-30-preview",
+        "Operation-Location": "https://accountName.purview.azure.com/share/operationResults/dad6baec-3a39-41df-a469-843a9ee94213?api-version=2022-06-30-preview",
+        "Operation-Id": "25c78f97-0b0a-4fe9-ad39-883a482265cd"
```
Comment (NateB2) [inline specification/purview/data-plane/Azure.Analytics.Purview.Share/preview/2022-06-30-preview/examples/ReceivedShares_AdlsGen2AccountUpdate.json]: The guid in the Operation-ID field should match the guid in the Azure-AsyncOperation and the Operation-Location field.

File: specification/purview/data-plane/Azure.Analytics.Purview.Share/preview/2022-06-30-preview/examples/ReceivedShares_AdlsGen2AccountGet.json [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    "receivedShareId": "0D67B9C8-A6C6-4990-9EDE-12EA059D3002"
+  },
+  "responses": {
+    "200": {
+      "headers": {
+        "Date": "Wed, 12 July 2022 18:04:32 GMT",
+        "x-ms-correlation-request-id": "25c78f97-0b0a-4fe9-ad39-883a482265cd"
+      },
+      "body": {
+        "id": "35E28F0E-DEA4-472F-84E4-5F1E45FB9937",
```
Comment (anniruddh) [inline specification/purview/data-plane/Azure.Analytics.Purview.Share/preview/2022-06-30-preview/examples/ReceivedShares_AdlsGen2AccountGet.json]: Good to match with id on line 5.

File: specification/purview/data-plane/Azure.Analytics.Purview.Share/preview/2022-06-30-preview/examples/ReceivedShares_Activate_TenantEmailRegistration.json [hunk 3]
```diff
@@ -0,0 +1,32 @@
+﻿{
+  "parameters": {
+    "endpoint": "accountName.purview.azure.com/share",
+    "api-version": "2022-06-30-preview",
+    "repeatability-request-id": "b470712a-ffbc-4d9f-ab99-6640db8bbcb8",
```
Comment (anniruddh) [inline specification/purview/data-plane/Azure.Analytics.Purview.Share/preview/2022-06-30-preview/examples/ReceivedShares_Activate_TenantEmailRegistration.json]: Should this be part of request header?

File: specification/purview/data-plane/Azure.Analytics.Purview.Share/preview/2022-06-30-preview/examples/ReceivedShares_Get.json [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+    "receivedShareId": "0D67B9C8-A6C6-4990-9EDE-12EA059D3002"
+  },
+  "responses": {
+    "200": {
+      "headers": {
+        "Date": "Wed, 12 July 2022 18:04:32 GMT",
+        "x-ms-correlation-request-id": "25c78f97-0b0a-4fe9-ad39-883a482265cd"
+      },
+      "body": {
+        "id": "35E28F0E-DEA4-472F-84E4-5F1E45FB9937",
```
Comment (anniruddh) [inline specification/purview/data-plane/Azure.Analytics.Purview.Share/preview/2022-06-30-preview/examples/ReceivedShares_Get.json]: Match with receivedShareId on line 5.

File: specification/purview/data-plane/Azure.Analytics.Purview.Share/preview/2022-06-30-preview/examples/ReceivedShares_GetAll_AttachedReceivedShares.json [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+                  "mountPath": "pathAbc"
+                }
+              }
+            },
+            "shareKind": "InPlace",
+            "type": "receivedShares"
+          }
+        ]
+      }
+    }
```
Comment (anniruddh) [inline specification/purview/data-plane/Azure.Analytics.Purview.Share/preview/2022-06-30-preview/examples/ReceivedShares_GetAll_AttachedReceivedShares.json:87]: Consider showing a nextLink as well in example.

File: specification/purview/data-plane/Azure.Analytics.Purview.Share/preview/2022-06-30-preview/examples/ReceivedShares_Register_TenantEmailRegistration.json [hunk 6]
```diff
@@ -0,0 +1,27 @@
+﻿{
+  "parameters": {
+    "endpoint": "accountName.purview.azure.com/share",
+    "api-version": "2022-06-30-preview",
+    "repeatability-request-id": "b470712a-ffbc-4d9f-ab99-6640db8bbcb8"
```
Comment (anniruddh) [inline specification/purview/data-plane/Azure.Analytics.Purview.Share/preview/2022-06-30-preview/examples/ReceivedShares_Register_TenantEmailRegistration.json]: In request header instead?

File: specification/purview/data-plane/Azure.Analytics.Purview.Share/preview/2022-06-30-preview/examples/SentShares_GetAll_SentShareInvitations.json [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+              "senderTenantName": "Test Tenant",
+              "senderEmail": "testSender@microsoft.com",
+              "provisioningState": "Succeeded"
+            },
+            "id": "DF4A2AAE-8755-47BB-9C00-A774B5A7006E",
+            "type": "sentShares/sentShareInvitations",
+            "invitationKind": "Service"
+          }
+        ],
+        "nextLink": "https://accountName.purview.azure.com/share/sentShareInvitations?api-version=2022-06-30-preview&skipToken=skip_token_to_fetch_more_sent_share_invitations"
```
Comment (anniruddh) [inline specification/purview/data-plane/Azure.Analytics.Purview.Share/preview/2022-06-30-preview/examples/SentShares_GetAll_SentShareInvitations.json]: sentShareId segment missing

**Review by anniruddh — COMMENTED** [review]
Few minor fixes required, will need to backport to Share repo as well.

**Review by NateB2 — CHANGES_REQUESTED** [review]

**Review by anniruddh — APPROVED** [review]

**Comment by faisalaltell** [comment]
This is just a starting point, for any other improvements feel free to create a PR into the main branch.

**Review by NateB2 — APPROVED** [review]

File: specification/purview/data-plane/Azure.Analytics.Purview.Share/preview/2022-06-30-preview/share.json [hunk 8]
```diff
@@ -0,0 +1,1598 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Purview Share Client",
+    "description": "Creates a data plane client for Purview Share.",
+    "version": "2022-06-30-preview"
+  },
+  "paths": {
+    "/v2/receivedShares/{receivedShareId}": {
```
Comment (anniruddh) [inline specification/purview/data-plane/Azure.Analytics.Purview.Share/preview/2022-06-30-preview/share.json:9]: Need to remove /v2

**Review by anniruddh — COMMENTED** [review]
Please remove /v2 from all routes in spec.

**Pull request merged**